lib/linguist/languages.yml

```diff
@@ -3224,11 +3224,16 @@ MoonScript:
   language_id: 238
 Motorola 68K Assembly:
   type: programming
-  group: Assembly
+  aliases:
+  - m68k
   extensions:
-  - ".X68"
+  - ".asm"
+  - ".i"
+  - ".inc"
+  - ".s"
+  - ".x68"
   tm_scope: source.m68k
-  ace_mode: assembly_x86
+  ace_mode: text
```

Comment:
Even when the languages have almost nothing in common?

Reply:
Well, [I'll let you see for yourself](https://ace.c9.io/build/kitchen-sink.html). In the sidebar, select `Assembly x86` from the `Mode` menu, and compare its sample's highlighting with that of Motorola assembly. Not perfect, but far from terrible.

A lot of languages have imperfect grammars or editor modes assigned, simply because it's a passable approximation. Motorola has comments and labels highlighted, and that's really a step up from absolutely no highlighting whatsoever...

Reply:
It's worth pointing out that Intel's is the only assembly dialect supported by ACE, meaning it may as well have the `x86` part removed and just serve as an umbrella for the editor's assembly support.

Reply:
Yes, an ACE syntax only supporting the most generic assembly directives like includes, sections, equations and comments would be most useful.

Reply:
Yes, a sarcastic quip about keyword recognition from a browser-based code editor will certainly get this PR approved and merged sooner.  <img src="https://user-images.githubusercontent.com/2346707/66113037-5535d500-e60f-11e9-80a6-287ae2ffc31a.gif">

Are we done here?

Reply:
Heh. Well, I'm used to seeing assembly incorrectly highlighted ([except when I'm not](https://github.com/github/linguist/pull/3112#issuecomment-232148398)), so I naturally can't see basic code-editors grokking full instruction sets for every known dialect of assembler. 😜

Anywho... if you could revert this line back to `ace_mode: assembly_x86` the way it was, that'd be copacetic. 👍

Reply:
Done.

Reply:
> Yes, an ACE syntax only supporting the most generic assembly directives like includes, sections, equations and comments would be most useful.

If the `x86` part is breaking syntax highlighting for other assembly dialects, that may make sense. In any case, it would require a pull request to Ace, not here.

In the meantime, unless there are cases for which the `assembly_x86` Ace mode is making it worse, I agree with @Alhadis and think it would be a good fallback compared to no highlighting at all.